src/app/main.py

```diff
@@ -22,6 +23,33 @@
 app = FastAPI(root_path=f"/{root_path}")
 
 
+def custom_openapi():
+    """Custom function to modify the OpenAPI schema and remove 422 errors."""
```

Comment:
Please add a link to https://github.com/fastapi/fastapi/discussions/6695, which is where this implementation seems to come from, and is also where additional context can be found (as without additional context, this code looks really hacky and suspicious).

Reply:
Yes, exactly. This is where I found the fix. I will add it to the code.